eksrollup/lib/k8s.py

```diff
@@ -271,8 +271,8 @@ def match_k8s_pods(pods, regex_compiled):
     matched_pods = []
     logger.info(f'Applying regex "{regex_compiled.pattern}" on')
     for pod in pods:
-        pod = dict(pod)
-        name = pod["metadata"]["name"]
+        p = pod.to_dict()
```

Comment:
Please do not use short forms of vars (`p`), use meaningful names instead

```suggestion
        pod_dict = pod.to_dict()
```